arho_feature_template/core/geotiff_creator.py

```diff
@@ -125,7 +125,8 @@ def _create_geotiff_from_png(self, image_path, geotiff_path):
             ds,
             outputSRS="EPSG:3067",
             format="GTiff",
-            creationOptions=["COMPRESS=LZW", "TILED=YES"],
+            outputType=gdal.GDT_Byte,
+            creationOptions=["TILED=YES", "COMPRESS=LZW", "PHOTOMETRIC=RGB", "BIGTIFF=IF_SAFER", "NBITS=8"],
```

Comment:
Ilmeisesti Ryhdin validointi suuttui eniten siitä, että kuva oli RGBA. Eli nuo lisätyt creation optionit taitaa olla turhia. GeoTiffin saa RGB:ksi bandList-parametrilla. Voisi kokeilla mitä validointi sanoo tällatavalla tehdystä GeoTiffistä:
```suggestion
            bandList=[1, 2, 3],
            creationOptions={
                "COMPRESS": "LZW",
                "TILED": "YES",
                "BIGTIFF": "IF_SAFER",
            },
```